frontend/app/src/components/ChallengeGrid/ChallengeItem.js

```diff
@@ -60,10 +60,10 @@ const ChallengeItem = ({ item }) => {
           }}
         >
           <div className="ChallengeItem--text">
-            <img class="ChallengeIco open" src={icon.url} />
+            <img className="ChallengeIco open" src={icon.url} />
```

Comment:
`className` is keyword for React components to define class. Normal html components like img, span and so on should use `class`

Reply:
This is actually a great clarification Visa! I've always wondered what's the difference and when to use which.

Reply:
Ah, true. I used search-replace in a bit too aggressive way. I think Kasper already fixed this.

Reply:
@Earlo I looked a bit into this. Shouldn't one use className when writing JSX? Also for html elements like div, img etc.